client/src/tables/orgEmployeeRegion.js

```diff
@@ -136,6 +136,7 @@ export default {
       var groups = _.groupBy(
         this.data,
         function (x) {
+          console.log(x);
```

Comment:
Woops

Reply:
Yup, I have it fixed, just gonna finish this employee.js file before i push